WcaOnRails/db/migrate/20180526084857_set_registration_requirements_for_upcoming_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class SetRegistrationRequirementsForUpcomingCompetitions < ActiveRecord::Migration[5.1]
+  def up
+    Competition.not_over.visible.where(registration_requirements: nil).update_all(registration_requirements: "* *")
```

Comment:
Actually, wouldn't it be better to do this for old comps as well? (Just to make them valid)

Reply:
For all `visible` ones then?

Also rubocop is complaining that I have inserted a non breaking space in there, which is perfectly intentional, as inserting just a space is rendered as a bullet :(

Reply:
Yeah, all `visible` sounds good.
To workaround the error you can use other text, e.g. `"_ _"` or `"> "`.